### PR TITLE
feat: Release notes for Codacy Cloud February 2024 DOCS-631

### PR DESCRIPTION
**Pull request by claudiacarpinteiro**
Adds auto-generated release notes for Codacy Cloud February 2024

### :eyes: Live preview
https://feat-release-notes-cloud-2024-02--docs-codacy.netlify.app/release-notes/cloud/cloud-2024-02/

### :construction: To do
- [x] Add the new release notes page to `mkdocs.yml` and `docs/release-notes/index.md`
- [x] Review generated release notes
- [x] Review list of issues with missing release notes
- [x] Review links to changelogs of updated tools
- [x] Update release date and remove `TODO` comments

### CODE REVIEW

File: docs/release-notes/cloud/cloud-2024-02.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   The Codacy GitHub App no longer requires **Read and write access to Administration** repository permission. For more information, see the [Discontinuation of SSH keys for GitHub repositories](./cloud-2024-01-15-gh-repository-ssh-keys-discontinuation.md). (PLUTO-803)
+-   The coverage data sent by the new Coverage engine to your Git provider moved from **beta** to stable. Thus, the coverage data sent by the old engine is now marked **deprecated**. For more information, [see the rollout details of the new Coverage engine](./cloud-2023-11-23-new-coverage-engine-status-checks.md). (ALA-766)
+-   The [Coverage Pull Requests page](../../repositories-coverage/pull-requests.md) now gets data from the new Coverage engine. For more information, [see the rollout details of the new Coverage engine](./cloud-2023-11-23-new-coverage-engine-status-checks.md). (ALA-770)
+-   The [Organization overview](../../organizations/organization-overview.md) is now available for the open source plan. (ALA-842)
```
Comment (nicklem) [inline docs/release-notes/cloud/cloud-2024-02.md:21]: Should we mention "on all plans" as an inclusive version? Or could it confuse some users?

```suggestion
-   The [Organization overview](../../organizations/organization-overview.md) is now available on all plans. (ALA-842)
```

File: docs/release-notes/cloud/cloud-2024-02.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+-   Improved the mechanism to refresh organizations from the Git provider. As result, we removed the option to manually refresh the list of organizations when [adding new organizations to Codacy](../../organizations/what-are-organizations.md#adding-an-organization). (ALA-862)
```
Comment (nicklem) [inline docs/release-notes/cloud/cloud-2024-02.md:22]: Nitpick:

```suggestion
-   Improved the mechanism to refresh organizations from the Git provider. As a result, we removed the option to manually refresh the list of organizations when [adding new organizations to Codacy](../../organizations/what-are-organizations.md#adding-an-organization). (ALA-862)
```

File: docs/release-notes/cloud/cloud-2024-02.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+Bugs and other issues:
+-   https://codacy.atlassian.net/browse/PLUTO-900
+
+-->
+
+## Product enhancements
+
+-   The Codacy GitHub App no longer requires **Read and write access to Administration** repository permission. For more information, see the [Discontinuation of SSH keys for GitHub repositories](./cloud-2024-01-15-gh-repository-ssh-keys-discontinuation.md). (PLUTO-803)
+-   For GitHub, if you have email privacy enabled on the provider, Codacy doesn't show your private email addresses when displaying your personal details on lists, search filters, or activity. (PLUTO-794)
+-   Added support for the Checkstyle filter [SuppressionCommentFilter](https://checkstyle.sourceforge.io/filters/suppressioncommentfilter.html) to suppress audit events. (TCE-709)
```
[the remark this answers is not in the document]
Comment (codacy-vrhpires) [reply docs/release-notes/cloud/cloud-2024-02.md:24]: Also:
- SuppressWarningsHolder
- SuppressWarningsFilter

[the remark this answers is not in the document]
Comment (claudiacarpinteiro) [reply docs/release-notes/cloud/cloud-2024-02.md]: Thanks, @codacy-vrhpires! Updated! 👍

**Review by machadoit — APPROVED** [review]
LGTM! :D

**Pull request merged**